rc/extra/tmux-repl.kak

```diff
@@ -5,7 +5,7 @@ hook global KakBegin .* %{
     %sh{
         if [ -n "$TMUX" ]; then
             VERSION_TMUX=$(tmux -V)
-            VERSION_TMUX=$(expr "${VERSION_TMUX}" : 'tmux \([0-9]*\|master\)')
+            VERSION_TMUX=$(expr \( "${VERSION_TMUX}" : 'tmux \([0-9]*\)' \) \| \( "${VERSION_TMUX}" : 'tmux \(master\)' \))
```

Comment:
that starts to be pretty complex, what we want to get is whatever is before the first dot of the second word. Can you try to just use something like:
```
VERSION_TMUX=$(tmux -V | cut -d' ' -f2)
VERSION_TMUX=${VERSION_TMUX%%.*}
```